### DISCUSSION TITLE
Select content width why not auto set as per button width ?

### DISCUSSION THREAD

**Discussion by akshata456** [spartan-ng/spartan#178] [Q&A]
<img width="1214" alt="Screenshot 2024-02-29 at 12 46 59 PM" src="https://github.com/goetzrobin/spartan/assets/34393874/28590b98-bce2-44e1-8869-e2655027b7e5">
here i want to set the select content container width will be set as per the parent div, but it not working.lmk what do here. thanks

**Comment by thatsamsonkid** [comment]
@akshata456 - This is a good point! Do you mind opening an issue with this. One thing you can do right now in the mean time is you can set the trigger to a specific width and also set the width or min-width on the hlm-select-content element to match the trigger. But I do think we should at least match the trigger width automatically. Will need to see how best to do that but should definitely be possible
